main/src/library/List.flix

```diff
@@ -262,10 +235,10 @@ namespace List {
     /// Returns `xs` with the element at index `i` replaced by `a`.
     /// Returns `xs` if `i < 0` or `i > length(xs)-1`.
     ///
-    def replace[a](i: Int32, a: a, xs: List[a]): List[a] = match (i, xs) with {
+    def update[a](i: Int32, a: a, xs: List[a]): List[a] = match (i, xs) with {
```

Comment:
I know you mentioned `substitute` or `swap` for this.  What do you think about `update`?  Scala has an almost identical function called `updated`.